webdriver/tests/bidi/script/add_preload_script/add_preload_script.py

```diff
@@ -235,3 +239,81 @@ async def add_preload_script_of_type(type):
         expected_result["value"].append({"type": "string", "value": type})
 
     assert result == expected_result
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize("create_type", ["popup", "iframe"])
+@pytest.mark.parametrize("access_type", ["evaluate", "evaluate2", "opener", "opener2", "inline"])
```

Comment:
It would be good to get some better names for the individual parameterizations. like `current_context_with_url`, `current_context_without_url`, `opener_context_with_url`, ..., `data_url`. Then it will be easier to understand when seeing test failures in CI.

Also lets exchange the order so that the creation type is listed first in the test name (it picks the latest parameterization first).

```suggestion
@pytest.mark.parametrize("access_type", ["evaluate", "evaluate2", "opener", "opener2", "inline"])
@pytest.mark.parametrize("create_type", ["popup", "iframe"])
```